src/Boleto/AbstractBoleto.php

```diff
@@ -1096,7 +1096,7 @@ public function getJuros()
      */
     public function getMoraDia()
     {
-        if (!$this->getJuros() > 0) {
+        if (!$this->getJuros() < 0) {
```

Comment:
aqui se o juros for > 0 sempre vai retornar 0 nesse caso nunca vai retornar mora

Reply:
tem que tirar esse "!" para tirar o bug e nao tirar o > 0, o correto seria 

if ($this->getJuros() <= 0) {
     return 0;
}

pq esse **!**$this->getJuros() vai sem dar o bug nao importa o <0 ou >0 depois